types/request.d.ts

```diff
@@ -1,3 +1,4 @@
+import { ErrorObject } from 'ajv'
```

Comment:
This is a transitive import
Is it possible to use the types exported by `@fastify/ajv-compiler`?

If it is not possible I think we should export them from there first.
This is necessary to use the right ajv version

Reply:
I tried - I can't get the `errorObject` type from `@fastify/ajv-compiler` :(

Can anyone else try?

Reply:
I dont think its possible to infer the ErrorObject. I guess you should create a PR in ajv compiler and export the ErrorObject.

Reply:
Have created https://github.com/fastify/fastify/pull/4283

Reply:
```suggestion
import { ErrorObject } from '@fastify/ajv-compiler'
```